apps/notepad/src/app/component/note-details/note-details.component.html

```diff
@@ -0,0 +1,56 @@
+<form [formGroup]="createNoteDetailForm">
+  <div>
+    <div class="flex">
+      <div class="max-w-sm rounded shadow-lg">
+        <input
+          class="text-center font-extrabold text-2xl"
+          type="text"
+          name="inputTitle"
+          formControlName="detailTitle"
+          [disabled]="!isEditable"
+        />
+        <input
+          type="text"
+          name="inputContent"
+          formControlName="detailContent"
+          [disabled]="!isEditable"
```

Comment:
Avoid using disabled attribute on input element in html when using Reactive forms.